.gitignore

```diff
@@ -408,3 +414,6 @@
 
 # /utils/wxrc/
 /utils/wxrc/*.sln
+
+# /include
+/include/**/*.gch
```

Comment:
i will try to check the mingw with `-include` if it could use the gch from different path wih `.h` folder